vulnerabilities/api.py

```diff
@@ -690,3 +690,112 @@ class AliasViewSet(VulnerabilityViewSet):
     """
 
     filterset_class = AliasFilterSet
+
```

Comment:
What about putting all the v2-related code in a new api_v2 module to ease the maintainability?